src/crimson/os/seastore/omap_manager.h

```diff
@@ -71,6 +71,13 @@ class OMapManager {
     const std::string &key,
     const ceph::bufferlist &value) = 0;
 
+  using omap_set_keys_ertr = base_ertr;
+  using omap_set_keys_ret = omap_set_keys_ertr::future<>;
+  virtual omap_set_keys_ret omap_set_keys(
+    omap_root_t &omap_root,
+    Transaction &t,
+    const std::map<std::string, ceph::bufferlist>& keys) = 0;
```

Comment:
instead of passing a const reference, i think we should pass a rvalue reference. so the callee can *move* from `keys` without worrying the overhead of memcpy.